libtenzir/src/curl.cpp

```diff
@@ -126,8 +126,10 @@ auto easy::set_http_header(std::string_view name, std::string_view value)
       break;
     }
   }
-  auto header = fmt::format("{}: {}", name, value);
-  headers_.append(header);
+  if (not value.empty()) {
+    auto header = fmt::format("{}: {}", name, value);
+    headers_.append(header);
+  }
```

Comment:
Did this bite you?

The reason why empty values were supported was to make it possible to *unset* header values. It's a cURL idiom to use an empty value for that. This ability is now gone.

Reply:
Yes; the fix for this doesn't work either, though, so the reason why we're unable to override the content type likely lies elsewhere. We temporarily worked around this with `--octet-stream`, but that should go away again.